packs/st2/actions/workflows/call_home.yaml

```diff
@@ -0,0 +1,66 @@
+---
+  vars:
+    bucket: "st2express-install"
+
+  chain:
+    -
+      name: "check_for_permission_to_send_anonymous_data"
+      ref: "st2.kv.get"
+      params:
+        key: "st2::collect_anonymous_data"
+      on-success: "get_server_uuid"
+    -
+      name: "get_server_uuid"
+      ref: "st2.kv.get"
+      params:
+        key: "st2::server_uuid"
+      publish:
+        server_uuid: "{{get_server_uuid.result}}"
+      on-success: "get_user_data"
+    -
+      name: "get_user_data"
+      ref: "st2.kv.get"
+      params:
+        key: "st2::user_data"
+      publish:
+        user_data: "{{get_user_data.result}}"
+      on-success: "generate_anonymous_data"
+    -
+      name: "generate_anonymous_data"
+      ref: "core.local"
+      params:
+        cmd: "st2-submit-debug-info --exclude-configs --exclude-logs --exclude-content --review"
```

Comment:
You should also use "--yes" flag so it runs in a non-interactive mode (otherwise it will prompt user for additional information with the latest in development version).